### PR TITLE
Transactions API missing data found in legacy (GWY-4835)

### PR DESCRIPTION
**Pull request by maxyengine**
Add child transactions to response

Resolves: GWY-4835

### CODE REVIEW

**Comment by XaosSintez** [comment]
Is that all missing data? Or is there something else?

**Comment by maxyengine** [comment]
The rest are missing in spec

File: spec/definitions/Transaction.yaml [hunk 1]
```diff
@@ -134,6 +140,12 @@ properties:
     description: Acquirer name, available only when method is 'payment_card', else null
     type: string
     readOnly: true
+  velocity:
+    description: Transaction's velocity
```
Comment (adamaltman) [inline spec/definitions/Transaction.yaml]: `The number of transactions by the same customer in the past 24 hours`

File: spec/definitions/Transaction.yaml [hunk 2]
```diff
@@ -134,6 +140,12 @@ properties:
...
+    type: number
+    format: float
```
Comment (adamaltman) [inline spec/definitions/Transaction.yaml]: is `integer` not an option?

File: spec/definitions/Transaction.yaml (hunk 1 above)
Comment (maxyengine) [inline spec/definitions/Transaction.yaml]: thanks

File: spec/definitions/Transaction.yaml (hunk 2 above)
Comment (maxyengine) [inline spec/definitions/Transaction.yaml]: right, i'll fix it

**Pull request merged**